src/theme/components/template.ts

```diff
@@ -1,113 +1,68 @@
 import { defineStyleConfig } from "@chakra-ui/react";
 import { defineStyle } from "@chakra-ui/system";
-/**
- * Grid layout based on https://www.joshwcomeau.com/css/full-bleed/
- */
+
+export const responsiveGap = { base: "1rem", md: "1.5rem", xl: "1rem" };
+export const responsivePadding = { base: "1rem", md: "1.5rem", xl: "2rem" };
 
 const Template = defineStyleConfig({
   baseStyle: defineStyle({
     boxSizing: "border-box",
     color: "ui.typography.body",
     display: "grid",
-    gridTemplateColumns: `
-      1fr
-      min(1280px, 100%)
-      1fr`,
-    rowGap: "grid.l",
-    _dark: {
-      color: "dark.ui.typography.body",
-    },
-  }),
-  sizes: {},
-  defaultProps: {},
-});
-// Elements that need to breakout will span outside
-// the center 1280px grid column.
-const TemplateBreakout = defineStyleConfig({
-  baseStyle: defineStyle({
-    width: "100%",
-    // This could be "1 / 4" and it would mean the same. This is
-    // "future-proof" the grid column assignment to the last column.
-    gridColumn: "1 / -1",
+    maxWidth: "1280px",
+    m: "0 auto",
+    p: "s",
+    gridTemplateAreas: `"breakout" "top" "main" "bottom"`,
+    gridTemplateColumns: "repeat(1, minmax(100px, 1fr))",
```

Comment:
How did you get 100px for the first value in minmax?

Reply:
Arbitrarily.... it should probably be decided on by @bigfishdesign13.

Reply:
The smallest screen width that designers consider (in theory) is 320px, so I'd suggest changing the min here to 320px.

Why use `repeat` when it's not repeating?

Reply:
Actually, I'd argue that `minmax` should be removed altogether from the column styles and instead apply `minWidth: "320px"` to the full grid.